sql/47_raid_stats_form_id_and_pk_update.up.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE raid_stats
+    ADD COLUMN form_id SMALLINT UNSIGNED NOT NULL DEFAULT 0 AFTER pokemon_id;
+
+ALTER TABLE raid_stats
+    DROP PRIMARY KEY;
+
+ALTER TABLE raid_stats
+    ADD PRIMARY KEY (pokemon_id, date, level, form_id);
```

Comment:
Previously this was `date`,`area`,`fence`,`pokemon_id` so that there was a natural date order, would recommend that it is now `date`,`area`,`fence`,`pokemon_id`, `form_id`